src/test/ReactTestUtils.js

```diff
@@ -384,6 +384,7 @@ var NoopInternalComponent = function(element) {
   this._renderedOutput = element;
   this._currentElement = element;
   this._debugID = nextDebugID++;
+  ReactInstrumentation.debugTool.onInstantiateComponent(this._debugID, element);
```

Comment:
I'm thinking https://github.com/facebook/react/issues/7240#issuecomment-236403307 and further reports. But not sure if that was related or not.